### ISSUE TITLE
Does batch size influence the effect of training?

### ISSUE THREAD

**Issue by soldatjiang** [AlexeyAB/darknet#886]
I don't have enough GPU memory to set batch_size to 64.

**Comment by AlexeyAB** [comment]
What GPU do you use? 

`GPU memory usage = batch / subdivision`

https://github.com/AlexeyAB/darknet#how-to-improve-object-detection

**Comment by TheMikeyR** [comment]
yes, but you can increase the subdivision if you don't have enough gpu memory, e.g. 
batch=64
subdivision=32

**Comment by soldatjiang** [comment]
@AlexeyAB @TheMikeyR My GPU is GTX 1080, I increase subdivision to 16 and it works, thank you.
